packages/server/tests/acceptance/rpc_batch2.spec.ts

```diff
@@ -119,11 +126,11 @@ describe('@api-batch-2 RPC Server Acceptance Tests', function () {
         it('@release should execute "eth_estimateGas" for contract call', async function() {
             const expectedRes = `0x${(400000).toString(16)}`;
             const estimatedGas = await relay.call(RelayCalls.ETH_ENDPOINTS.ETH_ESTIMATE_GAS, [{
-                to: mirrorContract.evm_address,
+                to: basicContract.evm_address,
                 data: BASIC_CONTRACT_PING_CALL_DATA
             }], requestId);
             expect(estimatedGas).to.contain('0x');
-            expect(Number(estimatedGas)).to.be.gt(0);
+            expect(estimatedGas).to.equal(EthImpl.defaultTxGas || expectedRes);
```

Comment:
Q: Why  are we expecting 2 different values.
If the test passes or fails for one of them we won't know why.
Can we break this out to be very clear when the test expects one over the other. 
Maybe add a comment also.

Reply:
Both `EthImpl.defaultTxGas` and `expectedRes` seem to have the same value, maybe one of two can be removed.

Reply:
Yes, they are the same.  That was added in a previous refactor, but I'll remove one.